src/EventKit/EKEnums.cs

```diff
@@ -75,6 +75,16 @@ public enum EKErrorCode : long {
 		ProcedureAlarmsNotMutable,
 		EventStoreNotAuthorized,
 		OSNotSupported,
+		[iOS (13,0), Mac (10,15), Watch (6,0)]
```

Comment:
`EKErrorCode` is an error enum so it is unlikely that you will have to use its values as input so you can remove all the availability information (`[iOS (13,0), Mac (10,15), Watch (6,0)]`) from all the new enum members.